### PR TITLE
W3C Solid Community Group Meeting Guidelines

### PR DESCRIPTION
**Pull request by csarven**
Some notes below. Not required reading for the PR.

Using the W3C Process as guide, the recommendation is that the minutes of distributed meetings should be available after 48 hours. As far as I can tell from documentation related to scribing and minutes - and personal experience - groups come to an agreement about what's on record - how exactly they agree on the record is not specified. W3C groups use different approaches (and tools), e.g., minutes are shortly available after the meeting, draft minutes are generated, and approved by the group, and in other ways. I've pinged people as well as W3C (on IRC) for pointers and experiences but so far it seems that there is no formal process for agreeing to what has been minuted, and corrections are rare. Will share info once I know more.

Edit: The initial feedback that I got and reconfirmed:

* CSS WG posts the text of the official minutes to their mailing list as is (single URL). They allow in-meeting corrections (on IRC). Members are invited to reply with corrections in a new thread, and apparently that rarely happens.

* Credentials CG posts a copy of the text of official minutes to their mailing list and to a separate URL (for 8 years now), and apparently corrections were made only a few times (due to confidentiality reasons).

---

Minutes committed directly to the main branch takes the position that there is one URL for the minutes, scribed content is in sufficient quality and done in good faith, and allows corrections to be PR'd.

Corrections and missing information to the minutes are raised as PRs. Verifying changes to the minutes are done by the group. Relies on group's memory (and good faith) - no audio/video recordings for additional checks.

Concerns about prior group agreements are generally raised as a topic in meetings.

It'd be ideal to use only one URL to the minutes that's available shortly after the meeting. This has the same approach as what's commonly known as the "latest published version" in specs or "original resource" in Memento. It is useful to know the publication status of any documentation. Corrections are welcome and should be transparent. I do not ultimately what branch or whether there is a PR that a human/machine needs to merge.

### CODE REVIEW

**Comment by csarven** [comment]
Updated text in first comment https://github.com/solid/specification/pull/319#issue-754703186

**Comment by kjetilk** [comment]
I'm not going to flag any strong opinions on this, but also since silence doesn't forward anything, I'll just share my general preference. My general preference is to use the mechanics available to us at the platform we happen to be using, and therefore, I like the direction proposed by @bblfish .

**Comment by csarven** [comment]
Reasons behind the decisions are getting lost. Incomplete comparisons.

Some of the things that's relevant:

* Whether publication status is required or not. If not, a solution does not need to factor that in.
* Whether i) initial PR is required and ii) merging is on hold until group approval iii) in a (weekly) meeting.
* Whether the initial/final version of the minutes is in sufficient quality available from a single URL.
* Whether the announcements of the draft and/or final minutes need to be handled by existing tools.

It is not difficult to come up with workable solutions if considerations like the above have clear responses and there is unanimous or majority agreement. Clearly the assumptions and principles differ, and that's precisely why this PR exists, to have an actual process/guideline that we can use and improve.

Besides that, I don't find the comparisons particularly meaningful.

Arguing that x has worked for so long completely ignores the fact that it has also used up a lot of group time. Is that part of the base cost of running meetings and publishing minutes? Probably yes. It is worthwhile to not overlook the fact that some groups have also worked just fine without going through an explicit draft process.. for nearly a decade. Compare that to one panel's "9 months"! And, so the approach to push directly to main and improve later attempted to address that. But if the "requirement" is to get "approval" in a weekly meeting among those that are present in that particular meeting, then there is absolutely no point in talking about an approach that doesn't include that. Don't subscribe to some preferred requirements and then rule out the other solutions. They just have a different starting point. Having a draft branch still does not eliminate the issue on open PRs that may or may not be merged on a weekly basis - and we have data showing that. What's addressing that? Nothing is mentioned about that in the draft-minutes solution as if the problem just disappeared or didn't exist.

Essentially it all comes back down to 1) quality of transcripts 2) maintaining good faith 3) maturity of the group.

Not opposed to the solutions put forward, but we need to acknowledge why they all came forward and how they differ in their requirements and principles.

---

Suggestion to compromise/revise the proposal in this PR:

Create a PR (in whatever branch it is most suitable) immediately after meeting.
Participants have 24 (or 48) hours to provide improvements.
Then after an assignee (scribe, moderator, CG chair...) merges. This could be automated.

I hope we can compromise and reach consensus. We've reached our monthly bikeshedding quota :) Jokes aside, we needed to have this discussion because we needed to improve current process/practice. So, that's important IMO.)

**Comment by csarven** [comment]
>So finally we come to your telling us what the problem you were hoping to solve was. Because until now I have never heard anyone bring this up.

Not news. I was quite clear a number of times in chats, PRs (including this one) about the time that goes into updating minutes. Let me know if you want me to fetch links and re-quote, again.

Others not bringing up does not mean that the issue did not exist. People go with the flow, have different thresholds,.. It is not always obvious up front until attention is drawn to it. If you don't think threads of minute updating is an issue, then that should be acknowledged up front. I'm acknowledging that time can be put into updating but there should be a limit because there are other things to consider as well. See again the last suggestion I made.

(You count GitHub triggering notifications for PR open/close as a plus, but do not acknowledge the cost / cognitive load on getting notifications for each update in your argument/calculation.)

>perhaps give a soft deadline for PRs to occur before Saturday evening (our meetings are on Wednesday)

Perhaps. We can't assign people to do stuff on a weekend, but it can be automated.

I've mentioned several times now that the authorization-panel is not the only one that matter or even the places where any particular person is engaged in.

>have a process on ACTIONS

So let me get this straight. Cool. Who wants to volunteer? Until someone wants to assign themselves to write the code/docs around that, it is not worthwhile to consider. Perhaps in the future. We could also just go back to using W3C's stuff - which I'd be completely fine with.

People want simplicity and very loudly bring up their discomfort with issues on updating publication status from "draft" to "published" - mind you that the thing totalling two commits - but somehow cool with throwing more requirements, wishlists into the mix and that's fine? (Again, the suggestion I made in the previous comment is making even more compromises.. and okay to go with PRs.)

>It terms of time savings I could easily argue that your (@csarven) insisting on proposing this push to main proposal,

You could argue but then I'll have to tell you to not read every other line, and consider the possibility that I am taking key considerations into account and facilitate consensus.

You could argue but then I'll have to tell you to not read every other line, and consider the possibility that I am taking key considerations into account and facilitate consensus.

I am responding to your concerns with the PR, as well as the proposal you're making with incomplete comparisons. That is not "insisting". If the ideas can be improved, adopt new concerns,... great. What do you think I've been trying to do this whole time?

>Create a PR (in whatever branch it is most suitable) immediately after meeting.
>Participants have 24 (or 48) hours to provide improvements.
>Then after an assignee (scribe, moderator, CG chair...) merges. This could be automated.

**Comment by bblfish** [comment]
@csarven's latest simplified proposal:
> Create a PR (in whatever branch it is most suitable) immediately after meeting.
Participants have 24 (or 48) hours to provide improvements.
> Then after an assignee (scribe, moderator, CG chair...) merges. This could be automated.

I don't see why the PR should be committed to `main` before the next panel meeting. It solves no problem I know of. It only risks removing the check during the meeting to see if anyone has objections. 

The process I see should be the following simple process [as I proposed in issue/discussion 268](https://github.com/solid/authorization-panel/discussions/268).

1. Push minutes to the `draft-minutes` branch of relevant repository immediately after the call.
2. Improvements to the minutes MUST be PRs (can be done by editing the content on github)
3. After approval of minutes in next neeting, press github "Merge Pull Request" button

That is open, visible to all and simple, as it should be. It fosters what in epistemology is known as Common Knowledge [Stanford Encyclopedia of Philosophy entry](https://plato.stanford.edu/entries/common-knowledge/), that everybody knowing that everybody knows p. That is what is needed to move forward on consistent ACTIONS, etc.

File: meetings/README.md
Comment (dmitrizagidulin) [reply meetings/README.md:18]: I think we're overthinking this. Merging minutes to `main` is totally fine.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: meetings/template.md
Comment (elf-pavlik) [inline meetings/template.md:14]:
```suggestion

## Regrets
* [name](url)
```

If we intend to use it for RSVP it seems to make sense to include this as well.

**Comment by csarven** [comment]
As we currently have no consensus on meeting publication steps, I've decided to remove the section on "Steps to publishing minutes" in this PR. It is better to have this guideline in (merged) with the parts that we do have consensus on, FWIW. I acknowledge that we may need more experience/feedback on meeting publication steps, i.e., what works and what doesn't for different people and groups. We can incorporate that later on. Thanks all.

**Pull request merged**
